README.md

```diff
@@ -48,24 +49,6 @@ Please:
 |Request preimage |    | :heavy_check_mark: |   | `Hash` hash <br/> |
 |Unrequest preimage |    | :heavy_check_mark: |   | `Hash` hash <br/> |
 
-## Scheduler
-
-| Name        | Light | XL | Nesting | Arguments | 
-| :---------- |:------------:|:--------:|:--------:|:--------|
-|Schedule |    |   |   | `BlockNumber` when <br/>`OptionschedulePeriodBlockNumber` maybe_periodic <br/>`schedulePriority` priority <br/>`BoxCallOrHashOfT` call <br/> | 
-|Cancel |    |   |   | `BlockNumber` when <br/>`u32` index <br/> | 
-|Schedule named |    |   |   | `Vecu8` id <br/>`BlockNumber` when <br/>`OptionschedulePeriodBlockNumber` maybe_periodic <br/>`schedulePriority` priority <br/>`BoxCallOrHashOfT` call <br/> | 
-|Schedule after |    |   |   | `BlockNumber` after <br/>`OptionschedulePeriodBlockNumber` maybe_periodic <br/>`schedulePriority` priority <br/>`BoxCallOrHashOfT` call <br/> |
-|Schedule named after |    |   |   | `Vecu8` id <br/>`BlockNumber` after <br/>`OptionschedulePeriodBlockNumber` maybe_periodic <br/>`schedulePriority` priority <br/>`BoxCallOrHashOfT` call <br/> |
-
-## Babe
-
-| Name        | Light | XL | Nesting | Arguments | 
-| :---------- |:------------:|:--------:|:--------:|:--------|
-|Report equivocation |    |   |   | `BoxEquivocationProofHeader` equivocation_proof <br/>`KeyOwnerProof` key_owner_proof <br/> | 
-|Report equivocation unsigned |    |   |   | `BoxEquivocationProofHeader` equivocation_proof <br/>`KeyOwnerProof` key_owner_proof <br/> | 
-|Plan config change |    |   |   | `NextConfigDescriptor` config <br/> | 
-
```

Comment:
There were no supported extrinsics in these pallets so I removed them for clarity.